spec/support/models/discount.cr

```diff
@@ -0,0 +1,18 @@
+class Discount < BaseModel
+  skip_default_columns
+
+  table do
+    primary_key id : String, value_generator: ->{ Random::Secure.hex }
```

Comment:
`value_generator`

Better name suggestions are very welcome, I don't like this name

Reply:
I was mostly looking for something shorter, but this is good enough for now I guess.

Thanks for the feedback, I'll write more tests to touch all the same paths that the UUID tests touch

Reply:
I think `value_generator` is ok. Maybe a bit verbose, but you wouldn't want to use `value` since that would indicate the return value of this.

Does this work with the `autogenerated` option? Say you want a String primary key, but the value is generated from postgres.... Are you able to just skip this?